examples/fungible-token/test-contract-defi/src/lib.rs

```diff
@@ -10,7 +10,7 @@ use near_sdk::{
 
 const BASE_GAS: u64 = 5_000_000_000_000;
 const PROMISE_CALL: u64 = 5_000_000_000_000;
-const GAS_FOR_FT_ON_TRANSFER: Gas = Gas(BASE_GAS + PROMISE_CALL);
+const GAS_FOR_FT_ON_TRANSFER: Gas = Gas::from_gas(BASE_GAS + PROMISE_CALL);
```

Comment:
```rust
const BASE_GAS: Gas = Gas::from_tgas(5);
const PROMISE_CALL: Gas = Gas::from_tgas(5);
const GAS_FOR_FT_ON_TRANSFER: Gas = Gas::from_gas(BASE_GAS.as_gas() + PROMISE_CALL.as_gas());
```

Reply:
I can't do that. Those two must be consts too, but  they are not.

Reply:
@iho What exactly is not const? `from_gas` and `as_gas` are const-fns